client/layout/logged-out.jsx

```diff
@@ -11,11 +11,8 @@ import { connect } from 'react-redux';
  * Internal dependencies
  */
 import MasterbarMinimal from 'layout/masterbar/minimal';
-import ThemesHead from 'my-sites/themes/head';
-import ThemeSheetComponent from 'my-sites/themes/sheet';
 
-const LayoutLoggedOutDesign = ( { routeName, match, section, hasSidebar, isFullScreen, tier = 'all' } ) => {
-	const primary = routeName === 'themes' ? <ThemeSheetComponent themeSlug={ match.theme_slug } /> : null;
+const LayoutLoggedOut = ( { primary, secondary, tertiary, section, hasSidebar, isFullScreen } ) => {
```

Comment:
The defaults are in `state/ui/reducer` or something. Don't see any harm in expressing them here too, though.